docs/opcodes/F7.mdx

```diff
@@ -0,0 +1,50 @@
+---
+fork: Prague
+group: Environmental Information
+---
+
+*Index 1 is top of the stack. See [PUSH](/#60).*
+
+## Notes
+
+A sub context can be created with [EXTCALL](/#F8), [EXTDELEGATECALL](/#F9) or [EXTSTATICCALL](/#FB). 
+
+The **RETURNDATALOAD** is used to load 32 bytes word from return data, and push it onto the memory stack, if return data is less than 32 bytes, the value is zero-padded (same behavior as [CALLDATALOAD](/#35)).
+
+## Stack input
+
+0. `offset`: 2-byte offset in the [return data](/about) from the last executed sub [context](/about) to copy.
+
+## Stack output
+
+0. `data`: 32-byte value starting from the given offset of the [return data](/about). All bytes after the end of the calldata are set to 0.
+
+## Examples
+
+| Returndata |
+|---------|
+| `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
+
+| * | Input | Output |
+|--:|------:|-------:|
+| `1` | `0x0000` | `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
+
+| Memory before | Memory after |
+|---------------|--------------|
+| `0x0000` | `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
```

Comment:
```suggestion
```

Afaik this doesn't modify the memory (only the stack is modified) 🤔 

([from the EIP](https://eips.ethereum.org/EIPS/eip-7069))

<img width="899" alt="Screenshot 2024-08-21 at 21 43 44" src="https://github.com/user-attachments/assets/1bf2a914-c2b8-4fb7-93e9-6cc3a10491c8">

Reply:
so the stack mean `return_stack`? I'm a bit confused between those two

Reply:
No, I'm pretty sure this is just referring to the normal EVM stack.

Below is the code from ehtereumjs, and `runState.stack` == `evm stack` (they have `runState.env.eof?.eofRunState.returnStack` used by the EOF return stack which is used by `CALLF`, `RETF` and `JUMPF` opcodes ) 

<img width="544" alt="Screenshot 2024-08-22 at 14 49 24" src="https://github.com/user-attachments/assets/39a54cee-68b7-4f43-adb5-5797426ed18f">